### ISSUE TITLE
After pressing show Printing Preview Print Dialog is not shown up, just blinking twice

### ISSUE THREAD

**Issue by David73230** [cleidigh/printing-tools-ng#260]
Mac OS High Sierra 10.13.6, Thunderbird 115.10.1, Printing Tools NG  v3.0.1

If i press Printing Preview, nothing happens, it is just blinking twice, no Printing Dialog is shown up.
I made a screenrecord. See here.

https://e.pcloud.link/publink/show?code=XZFImgZaTj3pkPKthboyV0NzSRkmHNCMAXk

**Comment by cleidigh** [comment]
@David73230 
Thank you for including the video, that made it clear what you were seeing.
This is tricky as my mac will only run up to Mojave.
Can you put these two words in debug options?
trace1 printsettings 

Then do a preview and capture and post the debug output here?
@cleidigh

**Comment by David73230** [comment]
Hello Christopher, thanks for your quick feedback. I have now put the two words into the debug options. However, I don't know where I can make this debug information visible. Maybe you know. I did a screen recording again. This is really stupid that I can't see the print preview. So I can't check the result of my printer settings. Could it be that it has something to do with this "Metal Graphics Engine" that Apple then installed in the newer Macs?

https://e.pcloud.link/publink/show?code=XZeWhgZur0hGQQfskYo3qQ7B0NhEHYjzq0k

**Comment by cleidigh** [comment]
@David73230 
Sorry, should not make assumptions...Need a wiki for this 
Open the debug console with Command-Shift-J 
Clear the console with the trash icon 
Return to Thunderbird window 
Try a preview
Return to the console 
Right click, Copy all messages 
Paste here 
@cleidigh

**Comment by David73230** [comment]
I hope i did it right
![Bildschirmfoto 2024-07-19 um 22 11 40](https://github.com/user-attachments/assets/dfb99758-19fb-4a4e-b362-194659749a23)

**Comment by David73230** [comment]
![Bildschirmfoto 2024-07-19 um 22 15 57](https://github.com/user-attachments/assets/d08d01e1-032c-4752-b7b0-be0ec5c2fe46)

**Comment by David73230** [comment]
[console-export-2024-7-19_22-19-52.txt](https://github.com/user-attachments/files/16316779/console-export-2024-7-19_22-19-52.txt)

**Comment by cleidigh** [comment]
@David73230 
You got it, thanks.
I don't see any ptng errors, however, this version does not have debug on the preview open. I have beta for Thunderbird v128 and v115 that does which might give us more clues. Before we try that, I did see a lot of errors and delays from other addons. Can you disable all others, restart Thunderbird and try again.
@cleidigh

**Comment by David73230** [comment]
I disabled all other addons.

[console-export-2024-7-20_11-22-7.txt](https://github.com/user-attachments/files/16319573/console-export-2024-7-20_11-22-7.txt)

**Comment by David73230** [comment]
The last stable Thunderbird Version, in my case, was 102.15.1
All Thunderbird Versions after that, are not working properly together with printing-tools-ng anymore.
Too bad, I got used to this tool that I will probably go back to this Thunderbird version. This is not bad, it is more important to me to have this print tool than the latest version of Thunderbird.

**Comment by David73230** [comment]
Hello Christopher, I have now reinstalled the old Thunderbird version. And I once sent you the debug information for comparison, maybe you can see something in direct comparison.

[console-export-2024-7-20_12-15-30-old-Thunderbird-stable.txt](https://github.com/user-attachments/files/16319769/console-export-2024-7-20_12-15-30-old-Thunderbird-stable.txt)

![Bildschirmfoto 2024-07-20 um 12 16 33](https://github.com/user-attachments/assets/e8a30109-7d82-400f-bd35-b0e3a4ed1677)

**Comment by David73230** [comment]
Hello Christopher, don't ask me, why, ### but now it works, Thunderbird has renewed itself again on the latest version, I had forgotten to switch off the automatic update. But now the print dialog is finally open. I do not get it. But be happy with me, it works and I'm happy. Thanks for your effort.

![Bildschirmfoto 2024-07-20 um 12 44 55](https://github.com/user-attachments/assets/1f182ee4-d9df-4cef-a705-4816892d5b0a)

**Comment by David73230** [comment]
I found the issue

![Uploading Bildschirmfoto 2024-07-20 um 12.59.50.jpg…]()

**Comment by cleidigh** [comment]
@David73230 
Excellent!
And you found the "Hide inline attachments" bug I already have a fix for in v3.1.0 beta. It had some tricky issues, I would be greatly appreciated if you could test that on osx. You can get it here : 
- #255 

@cleidigh

**Comment by David73230** [comment]
Many thanks, i have tried to install the adon, but Thunderbird 115.13.0 (64-Bit) give me that message, it can not installed.
It needs to be signed
![Bildschirmfoto 2024-07-23 um 19 48 30](https://github.com/user-attachments/assets/716b3352-1841-46e5-b01a-6be0b92a0bb5)

**Comment by cleidigh** [comment]
@David73230 
That's a new one. Addons are not signed so that is odd.
Will have to get back to you.
@cleidigh

**Comment by cleidigh** [comment]
@David73230 
Everyone is agreeing that this should not /Cannot happen. Just to be absolutely clear, you are installing into Thunderbird v115.13.0
No Firefox involved?
@cleidigh

**Comment by cleidigh** [comment]
There is the possibility that if you download with Firefox it might try to install it in Firefox. Try installing from Thunderbird addon tabgear icon/Install Add-on from file

**Comment by David73230** [comment]
Yes, I think you're right, it must have been Firefox. It worked now and it seems to be working, it only flashes "once" now but the print dialog opens and that's the most important thing. Thank you very much for your help! Where can I donate a small amount to you? I've been using Printingtools for years now and I just like it. Thank you for this tool. Have a nice day! Best wishes, David
new Sreenrecord
https://e.pcloud.link/publink/show?code=XZyHWgZWPQVqiPM2l5xkdSkaXBkCuaurplX

**Comment by cleidigh** [comment]
David 
Thank you very much for your complimentary words. Great you are a long time user! Can I put you on my list for discussion /help on new features or big releases?  No obligation just the opportunity to be involved.

The little flash you see occurs when the sysem setting for View attachments inline is opposite the PTNG setting. The flash is Thunderbird reloading the page after changing the setting. Glad it's working. BTW, the beta has the ability to move the headers and footers margins, sometimes these are very close to the paper edges.

There's a donate button on the addon page, thanks in advance!
https://addons.thunderbird.net/en-US/thunderbird/addon/importexporttools-ng/

Christopher

**Comment by David73230** [comment]
> Can I put you on my list for discussion /help on new features or big releases?
Yes you can.

Question: Is it possible to make the space between a little smaler?
![Bildschirmfoto 2024-07-24 um 19 12 08](https://github.com/user-attachments/assets/5d7ec3b7-0261-4c6b-9b03-fefa433921e1)

**Comment by David73230** [comment]
Ah Ok, i got it the big space is because the largest Word: In english "Attachment's"

**Comment by cleidigh** [comment]
Yes, when I do my major rewrite for web extensions I will finally make this more responsive in all locales.

**Comment by cleidigh** [comment]
@David73230 
I tightened up the headers in the de locale.
Will post beta shortly.
@cleidigh
